docs/Antora/modules/howto_guides/pages/howto_measure_single_value.adoc

```diff
@@ -0,0 +1,25 @@
+= Measure a single value
+
+[#measure_single_value]
+== Measure a single value
+
+Sometimes you just want to get a single measurement value of a signal and do not need a streaming session.
+This can be easily done by using the getLastValue() method:
+
+[tabs]
+====
+Cpp::
++
+[source,cpp]
+----
+void HowToDoc_MeasureSingleValue(daq::DevicePtr device)
+{
+    // get 1st signal of device...
+    auto signal = device.getSignalsRecursive()[0];
+    // ...and get a measurement value
+    float measurementValue = signal.getLastValue();
```

Comment:
Last value can be not assigned or have another type

Reply:
fair enough, my device delivered float so it was working fine. Adapted example